cob_bringup/drivers/bms.launch

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0"?>
 <launch>
+    <arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+    <arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+    <arg name="sim" default="$(arg sim)"/>
 
-	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
-	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
-
-	<node name="bms" pkg="cob_bms_driver" type="bms_driver_node" output="screen" clear_params="true">
-		<rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/bms.yaml"/>
-	</node>
-	<node ns="bms" name="power_state_aggregator" pkg="cob_bms_driver" type="power_state_aggregator.py" output="screen">
-			<remap from="power_state" to="/power_state"/>
-	</node>
-
+    <node name="bms" pkg="cob_bms_driver" type="bms_driver_node" output="screen" clear_params="true">
+        <rosparam command="load" file="$(arg pkg_hardware_config)/$(arg robot)/config/bms.yaml"/>
+    </node>
+    <node ns="bms" name="power_state_aggregator" pkg="cob_bms_driver" type="power_state_aggregator.py" output="screen">
+        <remap from="power_state" to="/power_state"/>
+    </node>
```

Comment:
I guess the two nodes above are `unless="$(arg sim)"` as they are not used in simulation, right? Put them in an `unless` `group`

Keep an empty line to the next `if="$(arg sim)"` for clearer structure